core/src/test/java/com/exadel/aem/toolkit/bundle/customlists/util/CustomListsHelperTest.java

```diff
@@ -0,0 +1,147 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exadel.aem.toolkit.bundle.customlists.util;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+import org.apache.sling.api.resource.Resource;
+import org.apache.sling.api.resource.ResourceResolver;
+import org.apache.sling.api.resource.ValueMap;
+import org.apache.sling.models.annotations.DefaultInjectionStrategy;
+import org.apache.sling.models.annotations.Model;
+import org.apache.sling.models.annotations.injectorspecific.ValueMapValue;
+import org.apache.sling.testing.mock.sling.ResourceResolverType;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+import com.exadel.aem.toolkit.bundle.customlists.models.GenericItem;
+
+import io.wcm.testing.mock.aem.junit.AemContext;
+import static org.junit.Assert.assertEquals;
+
+public class CustomListsHelperTest {
+
+    @Rule
+    public AemContext context = new AemContext(ResourceResolverType.RESOURCERESOLVER_MOCK);
+    private ResourceResolver resolver;
+    private static final String SIMPLE_LIST_PATH = "/content/aem-custom-lists/simpleList";
```

Comment:
Move constants above the local variables

Reply:
Done